src/utils/multiRun.js

```diff
@@ -122,7 +123,7 @@ const runInParallel = async (
   allJobs,
   { ignoreErrors, parallelLogs, parallelLimit, useTree }
 ) => {
-  const maxConcurrency = parallelLimit || Infinity;
+  const maxConcurrency = parallelLimit || Math.max(cpus().length - 1, 1);
```

Comment:
Note that `cpus` might lie, especially in docker or other virtualization environments. But it'd never be `Infinity` so should be fine